<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>drugAddDescription</class>
 <widget class="QMainWindow" name="drugAddDescription">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление/редактирование описания препарата</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="drugGroupTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>256</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Группа препаратов</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Название</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Идентификатор</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>120</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Описание</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="drugGroupSpec">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>371</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>260</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Лечение</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="drugGoupSick">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>290</y>
      <width>371</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="drugGroup">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="drugUseCode">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>80</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="clearToAddBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Очищает поля для добавления нового элемента</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/add.png</normaloff>icons/add.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteBtn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Удаляет выбранный элемент из таблицы</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/delete.png</normaloff>icons/delete.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="addDescriptionBtn">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>380</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Добавляет новый элемент в таблицу</string>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reloadBtn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>380</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/update.png</normaloff>icons/update.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>693</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
